### PR TITLE
Trends: small fixes

### PR DESCRIPTION
**Pull request by macobo**
## Changes

Trends logic sessions links had a few bugs which this irons out.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: frontend/src/scenes/insights/PersonModal.js [hunk 1]
```diff
@@ -34,7 +32,7 @@ export function PersonModal({ visible, view }) {
                     style={{ marginBottom: 16, display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}
                 >
                     Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
-                    {user?.is_multi_tenancy && featureFlags['filter_by_session_props'] ? (
+                    {featureFlags['filter_by_session_props_link'] ? (
```
Comment (paolodamico) [inline frontend/src/scenes/insights/PersonModal.js:35]: Context on why the switch in the feature flag?

Comment (macobo) [reply frontend/src/scenes/insights/PersonModal.js:35]: This way we can turn this feature on but not the new filters -> I think this is already useful as-is.

**Review by paolodamico — APPROVED** [review]
lgtm! out of the scope of this PR, but it's a bit confusing to go to the list of sessions and not see the filters applied (though they indeed are), and I think we can improve the UI of the filters in the modal a bit too.

**Comment by paolodamico** [comment]
Also worth noting as an improvement to add the #backTo link thing.

**Comment by macobo** [comment]
> lgtm! out of the scope of this PR, but it's a bit confusing to go to the list of sessions and not see the filters applied (though they indeed are), and I think we can improve the UI of the filters in the modal a bit too.

Hoping to get the new filters out early next week :crossed_fingers:

**Pull request merged**